daemon/api_users_test.go

```diff
@@ -140,43 +133,43 @@ func (s *userSuite) testCreateUser(c *check.C, oldWay bool) {
 		SSHKeys:          []string{"ssh1", "ssh2"},
 		OpenIDIdentifier: "xxyyzz",
 	}
-	osutilAddUser = func(username string, opts *osutil.AddUserOptions) error {
+	defer daemon.MockOsutilAddUser(func(username string, opts *osutil.AddUserOptions) error {
```

Comment:
(nitpick^2) I slightly prefer an explicit "restore := ...\ndefer restore()" here and it seems like that is what we mostly do elsewhere. But daemon seems to do it differently so that's fine to be consistent. Just ignore, definitely not important.

Reply:
I know that historically we did that, but I tend to prefer the more compact version nowadays.